NeoML/include/NeoML/Dnn/Layers/DropoutLayer.h

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 #include <NeoML/NeoMLDefs.h>
 #include <NeoML/Dnn/Dnn.h>
 #include <NeoML/Dnn/Layers/BaseInPlaceLayer.h>
+#include <NeoMathEngine/MathEngineDropout.h>
```

Comment:
Forward declaration is enough

Reply:
Need include to access "isValid" field in DropoutLayer.cpp file

Reply:
Why it cannot be included in `DropoutLayer.cpp` and in header to leave a forward declaration only?